src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/ArrivalsServiceConfirmArrivalTest.kt

```diff
@@ -312,36 +380,27 @@ class ArrivalsServiceConfirmArrivalTest {
     private val FIXED_CLOCK = Clock.fixed(FIXED_NOW, ZONE_ID)
     private val LOCATION = "RECP"
     private val INMATE_DETAIL = InmateDetail(
-      offenderNo = OFFENDER_NO, bookingId = BOOKING_ID,
+      offenderNo = PRISON_NUMBER, bookingId = BOOKING_ID,
       assignedLivingUnit = AssignedLivingUnit(
         PRISON_ID, LOCATION_ID, LOCATION, "Nottingham (HMP)"
       )
     )
     private val INMATE_DETAIL_NO_UNIT = InmateDetail(
-      offenderNo = OFFENDER_NO, bookingId = BOOKING_ID,
+      offenderNo = PRISON_NUMBER, bookingId = BOOKING_ID,
     )
     private val CONFIRM_COURT_RETURN_RESPONSE =
-      ConfirmCourtReturnResponse(prisonNumber = OFFENDER_NO, location = LOCATION, bookingId = BOOKING_ID)
+      ConfirmCourtReturnResponse(prisonNumber = PRISON_NUMBER, location = LOCATION, bookingId = BOOKING_ID)
     val ARRIVAL_PROTOTYPE = Arrival(
       id = MOVE_ID,
       firstName = FIRST_NAME,
       lastName = LAST_NAME,
       dateOfBirth = DATE_OF_BIRTH,
-      prisonNumber = null,
+      prisonNumber = PRISON_NUMBER,
       pncNumber = null,
       date = LocalDate.now(FIXED_CLOCK),
       fromLocation = "Kingston-upon-Hull Crown Court",
       fromLocationType = LocationType.COURT,
       isCurrentPrisoner = false,
-      potentialMatches = listOf(
-        PotentialMatch(
-          firstName = FIRST_NAME,
-          lastName = LAST_NAME,
-          dateOfBirth = DATE_OF_BIRTH,
-          prisonNumber = OFFENDER_NO,
-          pncNumber = null
-        )
-      )
```

Comment:
this wasn't valid as this object was being returned from BASM with matches, when matches are only added later... maybe we should represent this as two different types, before and after being enriched?